flink-tests/src/test/java/org/apache/flink/test/runtime/BlockingShuffleITCase.java

```diff
@@ -94,7 +93,6 @@ private JobGraph createJobGraph(int numRecordsToSend) {
 
         StreamGraph streamGraph = env.getStreamGraph();
         streamGraph.setGlobalDataExchangeMode(GlobalDataExchangeMode.ALL_EDGES_BLOCKING);
-        streamGraph.setScheduleMode(ScheduleMode.LAZY_FROM_SOURCES);
         // a scheduler supporting batch jobs is required for this job graph, because it contains
         // blocking data exchanges, and the "lazy from sources" schedule mode is not supported by
```

Comment:
comment still mentions the schedule mode

Reply:
True. I will correct it. Good catch.